packages/react-component-library/src/components/DataGrid/DataGrid.stories.tsx

```diff
@@ -247,13 +246,30 @@ Sorting.parameters = {
   },
 }
 
+const arbitraryData = [
+  ...data,
+  {
+    id: 11,
+    productName:
+      'Long product name: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+    quantity: 10,
+    price: '£673.00',
+  },
+]
+
+const StyledCell = styled.div`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`
+
 const columnsWithArbitraryContent: ColumnDef<object>[] = columns.map(
   (item) => ({
     ...item,
     cell: (info) => {
       const value = info.getValue() as string
 
-      return <Badge>{value}</Badge>
+      return <StyledCell>{value}</StyledCell>
```

Comment:
This has removed the Badge so it's no longer visually apparent there is arbitrary cell content:

![Screenshot 2024-05-30 at 08 33 25](https://github.com/Royal-Navy/design-system/assets/48086589/4991d552-adb4-4380-ba21-528dc09ce86f)

Should this styling maybe be applied via a column meta prop (`noWrap`?) and have a distinct story?

See example here used for alignment:

https://github.com/Royal-Navy/design-system/blob/9d5292ed941b3d8671f4d58d6b9dc724c14831d4/packages/react-component-library/src/components/DataGrid/DataGrid.d.ts#L6